source/ui.js

```diff
@@ -155,12 +167,35 @@ module.exports = async (options, pkg) => {
 	}
 
 	if (!hasCommits) {
-		const answers = await inquirer.prompt([{
-			type: 'confirm',
-			name: 'confirm',
-			message: 'No commits found since previous release, continue?',
-			default: false
-		}]);
+		const answers = await inquirer.prompt([
+			{
+				type: 'confirm',
+				name: 'confirm',
+				message: 'No commits found since previous release, continue?',
+				default: false
+			}
+		]);
+
+		if (!answers.confirm) {
+			return {
+				...options,
+				...answers
+			};
+		}
+	}
+
+	if (options.unknown) {
+		const answers = await inquirer.prompt([
+			{
+				type: 'confirm',
+				name: 'confirm',
+				when: isScoped(pkg.name) && options.publish && !pkg.private,
+				message: `This scoped repo ${chalk.bold.magenta(
```

Comment:
I'd rephrase this to `Failed to check availability of scoped repo name ${pkg.name}. Do you want to try and publish it anyway?`

Reply:
Amended this line with the new wording. 👍